Update plugin submission guidelines, serve preview images (#189)
vercel/hyper-site: pages/store/submit.mdx

## Create your plugin or theme

> You can find instructions and details about the process of creating a new plugin or theme in the [hyper repository](https://github.com/vercel/hyper/blob/canary/PLUGINS.md).

Reviewer: ```suggestion
You can find instructions and details about the process of creating a new plugin or theme in the [Hyper repository](https://github.com/vercel/hyper/blob/canary/PLUGINS.md).
```

## Publish your plugin or theme to npm

For users to easily install your package, we use npm as the underlying service, therefore your plugin or theme needs to be published to npm.
We suggest that your plugin is named with the prefix `hyper-` (for example `hyper-native`) and has either `hyper-plugin` or `hyper-theme` as a keyword depending on the type of your package.
This will help users easily identify your package as a Hyper plugin or theme.

Along with the previous suggestion, it'd be really helpful if your plugin is open-sourced so that the security conscious user will be able to review your code easily. To provide a repository in your `package.json` please refer to
[npm's documentation](https://docs.npmjs.com/files/package.json#repository).

## Submit your plugin or theme to Hyper Store

Create a pull request in [the hyper-site repository](https://github.com/vercel/hyper-site), in which you should edit the `plugins.json` file and add the following information:

```
{

>   // The example is written in the format expected in the file but it contains comments that do not go into the file but are just there to give more information on what you can provide.

Reviewer: ```suggestion
  // The example is written in the desired format. You do not need to include these comments.
```
